script/setup/install-runc

```diff
@@ -22,10 +22,11 @@ set -eu -o pipefail
 
 function install_runc() {
 	RUNC_COMMIT=$(grep opencontainers/runc "$GOPATH"/src/github.com/containerd/containerd/vendor.conf | awk '{print $2}')
+	RUNC_REPO=$(grep opencontainers/runc "$GOPATH"/src/github.com/containerd/containerd/vendor.conf | awk '{print $3}')
 
-	go get -d github.com/opencontainers/runc
+	git clone "$RUNC_REPO" /src/github.com/opencontainers/runc
```

Comment:
We need to check these scripts; I've been bit by similar approaches in docker/docker (given, those didn't use vendoring)

`go get -d` will use go modules to get the dependency, and therefore first get go modules from `master`, before switching to the selected  commit, which sometimes may cause unexpected results (see https://github.com/moby/moby/pull/41560)